sonic-net/sonic-buildimage
# [arm64][Nokia-7215-A1]Add support for Nokia-7215-A1 platform

#### Why I did it

 Add new Nokia build target and establish an arm64 build:

        Platform: arm64-nokia_ixs7215_52xb-r0
        HwSKU: Nokia-7215-A1
        ASIC: marvell
        Port Config: 48x1G + 4x10G



#### How I did it
1) Change make files for saiserver and syncd to use Bulleseye kernel
2) Change Marvell SAI version  to 1.11.0-1
3) Add Prestera make files to build kernel, Flattened Device Tree blob and ramdisk for arm64 platforms
4) Provide device and platform related files for new platform support (arm64-nokia_ixs7215_52xb-r0).

#### How to verify it
1) Build arm64 target for Nokia-7215-A1 with updated sonic-linux-kernel submodule including PR's https://github.com/sonic-net/sonic-linux-kernel/pull/311 and https://github.com/sonic-net/sonic-linux-kernel/pull/312
2) Run this image on arm64-nokia_ixs7215_52xb-r0 and verify all dockers are up and test basic commands like: 
      - show version
      - show platform summary
      - show platform syseeprom 
      - show platform fan
      - show platform psustatus
      - show platform firmware status
      - show platform temperature
      - show platform ssdhealth
3) Verify ports are up using "show interface status" command


#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
